py/mpconfig.h

```diff
@@ -739,4 +739,8 @@ typedef double mp_float_t;
 #define MP_UNLIKELY(x) __builtin_expect((x), 0)
 #endif
 
+// Use to refer to extern structs of which actual type is not known.
+// For instance 'extern struct _dummy_t mp_sys_stdin_obj;'
+struct _dummy_t;
```

Comment:
It's not really a configuration option.  I'd put it in misc.h.  And maybe name it _mp_dummy_t.